.github/workflows/build.yml

```
@@ -0,0 +1,21 @@
+name: build the package
+on: [pull_request]
```

```suggestion
on:
  - pull_request
```

Nit: I think we should have a consistent yaml-style.